runtimes/pythonrt/py.go

```diff
@@ -63,7 +63,27 @@ func parsePyVersion(s string) (major, minor int, err error) {
 	return
 }
 
+func isFile(path string) bool {
+	info, err := os.Stat(path)
+	if err != nil {
+		return false
+	}
+
+	return !info.IsDir()
+}
+
+const exeEnvKey = "AK_PYTHON"
+
 func findPython() (string, error) {
+	exePath := os.Getenv(exeEnvKey)
```

Comment:
1. I can't comment on the exact line because it's not new/changed, but there's a typo in the error at the bottom of this function:

   `"non of %v found in PATH"` --> `none of...`

2. Please document this function, unlike `isFile` it's not obvious what it should return

Reply:
Done.